packages/runtime-core/__tests__/componentProxy.spec.ts

```diff
@@ -0,0 +1,109 @@
+import { createApp, getCurrentInstance, nodeOps } from '@vue/runtime-test'
+import { PublicInstanceProxyHandlers } from '../src/componentProxy'
+
+describe('component: proxy', () => {
+  it('data', () => {
+    const app = createApp()
+    let ctx: any
+    const Comp = {
+      data() {
+        return {
+          foo: 1
+        }
+      },
+      mounted() {
+        ctx = getCurrentInstance()
```

Comment:
Can simply be `instanceProxy = this`, which is what *should* be tested. There's no need to create the proxy manually.